searchcore/src/tests/proton/documentmetastore/documentmetastore_test.cpp

```diff
@@ -1460,6 +1498,7 @@ TEST("requireThatBucketInfoIsCorrectAfterJoin")
     EXPECT_EQUAL(bi21.getDocumentCount() +
                  bi23.getDocumentCount(),
                  bi11.getDocumentCount());
+    f.assertNotifyCreateBuckets({ f.bid11 });
 }
 
 TEST("requireThatActiveStateIsPreservedAfterJoin")
```

Comment:
Consider adding check for join test case as well.

Reply:
this is the join test case.

Reply:
Sorry, I misread the diff :)